packages/wrangler/src/r2/create.ts

```diff
@@ -68,8 +69,14 @@ export async function Handler(args: HandlerOptions) {
 	logger.log(
 		`✅ Created bucket '${fullBucketName}' with${
 			location ? ` location hint ${location} and` : ``
-		} default storage class of ${storageClass ? storageClass : `Standard`}.`
+		} default storage class of ${storageClass ? storageClass : `Standard`}.\n`
 	);
+
+	logger.log("Configure your Worker to write objects to this bucket:\n");
```

Comment:
unless there is a specific reason to have these as separate calls, I'd suggest 👇 

```suggestion
	logger.log(
	  "Configure your Worker to write objects to this bucket:\n" +
	  "[[r2_buckets]]\n" +
	  `bucket_name = "${args.name}"\n` +
	  `binding = "${getValidBindingName(args.name, "r2")}"`
	);
```

Reply:
Done!